src/idl_gen_rust.cpp

```diff
@@ -1832,3 +1844,6 @@ std::string RustMakeRule(const Parser &parser, const std::string &path,
 // TODO(rw): Generated code should generate endian-safe Debug impls.
 // TODO(rw): Generated code could use a Rust-only enum type to access unions,
 //           instead of making the user use _type() to manually switch.
+// TODO(maxburke): There should be tests added that use language keywords as
```

Comment:
"there should be test schemas added" -- this makes it clear how these tests would be created. :-)